docs/undergrad.yaml

```diff
@@ -282,7 +282,7 @@ Single Variable Real Analysis:
   Sequences of real numbers:
     convergence: 'filter.tendsto'
     limit point: 'map_cluster_pt'
-    recurrent sequences: 'nat'
+    recurrent sequences: ''
```

Comment:
I don't understand why you removed that line. I agree it's a bit cryptic, but what else would you want to see here?

Reply:
The definition of a (linear) recurrent sequence, backed with the characteristic polynomial and the general solution. A recurrent sequence is much more than just a sequence.

Reply:
Oh, it's a rather uninteresting then, but okay.